product/charts/miq_reports/vm_chargeback.yaml

```diff
@@ -64,6 +64,8 @@ headers:
 - Day
 - Chargeback Rates
 - Fixed Compute Metric
+- Fixed Compute 1 Metric
+- Fixed Compute 2 Metric
```

Comment:
Where do these 2 new strings come from? Why were they added?

Reply:
They come from the table. I noticed that the other strings were there but not these two and figured that they may have needed to be added here.
<img src="https://user-images.githubusercontent.com/64800041/98376736-e85eec80-2011-11eb-8c58-6d6168805e53.png" width="500">

Reply:
`vm_chargeback.yaml` contains definition of a report, you cannot just add headers to it like this. Those headers would have to match some object attribute and this way the report would be just broken.

Reply:
alright I'll remove them

Reply:
Those 2 keys and their descriptions need to be added to `locale/en.yml` right about here: https://github.com/ManageIQ/manageiq/blob/master/locale/en.yml#L1346